### ISSUE TITLE
Integration

### ISSUE THREAD

**Issue by magwrap** [Kacper-Grzyb/rug-bum-bum#24]
Integrate all components:

- data,
- initialise the model -> hyperparameters,
- train, 
- predict, 
- convert to audio

Hyperparameter Management: Define all hyperparameters in one place (e.g., learning rate, sequence length, number of training epochs, hidden layer size).
